README.md

```diff
@@ -87,18 +87,18 @@ All released versions can be found [here](https://github.com/sonatype-nexus-comm
 | Hosted  |                      |
 | Group   |                      |
 
-NOTE: This is an early version of P2 Proxy and does not fully support the following as of yet:
+NOTE: p2 Proxy does not fully support the following as of yet:
 
-* [Composite Repositories](https://help.eclipse.org/neon/index.jsp?topic=%2Forg.eclipse.platform.doc.isv%2Fguide%2Fp2_composite_repositories.htm)
-  * This is implemented but likely will fail if the composite files reference absolute paths rather than relative paths
-* Old style (non-P2) update sites. The code that accomplished this can be found [here](https://github.com/sonatype/nexus-public/blob/nexus-2.x/plugins/p2/nexus-p2-repository-plugin/src/main/java/org/sonatype/nexus/plugins/p2/repository/proxy/P2ProxyRepositoryImpl.java)
+* Old style (non-p2) update sites. The code that accomplished this can be found [here](https://github.com/sonatype/nexus-public/blob/nexus-2.x/plugins/p2/nexus-p2-repository-plugin/src/main/java/org/sonatype/nexus/plugins/p2/repository/proxy/P2ProxyRepositoryImpl.java)
```

Comment:
Trivial: This isnot you but this sentence doesn't end in a period.

Reply:
Done 026b33e37278fdd285655fb647acc45dde7c1143